region/maridia/inner-green/East Sand Hall.json

```diff
@@ -814,18 +828,28 @@
         "canTrickyJump",
         "canTrickyUseFrozenEnemies",
         "canPlayInSand",
-        "canCrouchJump",
-        {"ammo": {"type": "PowerBomb", "count": 1}},
-        {"enemyDamage": {
-          "enemy": "Evir",
-          "type": "particle",
-          "hits": 1
-        }}
+        {"or": [
+          "canCameraManip",
+          {"and": [
+            {"ammo": {"type": "PowerBomb", "count": 1}},
+            {"enemyDamage": {
+              "enemy": "Evir",
+              "type": "particle",
+              "hits": 2
+            }}
+          ]}
+        ]}
       ],
       "note": [
-        "From the left pillar, jump and place a PB to make the Evirs rise again, while being careful not to kill the close Evir.",
-        "Freeze the first Evir such that its bottom is only slightly lower than the top of the pillar.",
-        "Crouch jump onto the Evir, then freeze the second about 1.5 to 2.5 tiles lower. Jump off of the sand to get to the left door."
+        "If entering from the sandfall above, come in at the far left of the transition, with either a spin-jump or i-frames, to avoid the Evir shot;",
+        "land on the right side of the pillar to the left, being careful to not bring left Evir on camera.",
+        "If coming from the right side of the room, then moonwalk on the center pillar to scroll the camera as far right as possible, to avoid getting hit by an Evir shot;",
+        "alternatively, jump slightly to the left and freeze the Evir, then return back to the center pillar before jumping across the sand.",
+        "Freeze the right Evir and jump onto it.",
+        "Quickly move left and fire an angle-down shot to freeze the left Evir before it descends too far.",
+        "Jump onto the left Evir, back up to the right side of it to maximize space to build momentum, and jump to the left onto the sand and then onto the ledge.",
```

Comment:
Momentum doesn't transfer when you jump while underwater.
But I have seen signs of gaining ~a pixel like you get pushed forward slightly on the jump frame.